app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Union
+
+
+def cache(func: callable) -> callable:
```

Comment:
```suggestion
def cache(func: Сallable) -> Сallable:
```

Reply:
Can't apply the changes. When i use "callable" with capital "C" i get "NameError: name 'Callable' is not defined. Did you mean: 'callable'?"

Reply:
you should import it
`from typing import Callable`

Reply:
got it, thanks